src/server/worker/api_server.go

```diff
@@ -1860,17 +2120,30 @@ func (a *APIServer) writeStats(pachClient *client.APIClient, objClient obj.Clien
 	outputTree.Serialize(outputBuf)
 	statsBuf := &bytes.Buffer{}
 	statsTree.Ordered().Serialize(statsBuf)
+	// Merge datum stats hashtree
+	buf := &bytes.Buffer{}
```

Comment:
Similar to above, what does using a `bytes.Buffer` gain us here over writing directly to `objW` below?

Reply:
Same as above.